<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1311</width>
    <height>781</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>zizicross</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>659</y>
      <width>1121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>1101</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Add File </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="gaussianBlur">
        <property name="text">
         <string>gaussian blur</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="intensiveSharper">
        <property name="text">
         <string>top sobel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="sharper">
        <property name="text">
         <string>sharper</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="identityBtn">
        <property name="text">
         <string>identity</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="grayscale">
        <property name="text">
         <string>Grayscale</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="original">
        <property name="text">
         <string>original</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cust5x5">
        <property name="text">
         <string>Cust5x5</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="custKern">
        <property name="text">
         <string>Cust3x3</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="saveImg">
        <property name="text">
         <string>save</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>570</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>191</width>
       <height>51</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QSpinBox" name="spinBox">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>10</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1081</width>
      <height>451</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="screenLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetNoConstraint</enum>
     </property>
     <item>
      <widget class="QScrollArea" name="scrollArea">
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOn</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOn</enum>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QAbstractScrollArea::AdjustToContents</enum>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>1063</width>
          <height>433</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>480</y>
      <width>331</width>
      <height>171</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>311</width>
       <height>161</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3x3"/>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_4">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>480</y>
      <width>541</width>
      <height>161</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>521</width>
       <height>141</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5x5"/>
    </widget>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>480</y>
      <width>160</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCheckBox" name="mtEnabled">
       <property name="text">
        <string>Enable MT</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="gpuAccelEn">
       <property name="text">
        <string>GPU ACCEL</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1311</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuzizicross">
    <property name="title">
     <string>zizicross</string>
    </property>
   </widget>
   <addaction name="menuzizicross"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
